Backend/database/firestore/utils/firestore_translator.py

```diff
@@ -0,0 +1,26 @@
+from google.cloud.firestore_v1 import DocumentSnapshot
+from pydantic import BaseModel
+from typing import Type, TypeVar
+
+T = TypeVar("T", bound=BaseModel)
+
+
+class FirestoreTranslator:
+    @staticmethod
+    def document_to_object(doc: DocumentSnapshot, obj_model: Type[T]) -> T:
+        obj_data = doc.to_dict()
+        obj_data['id'] = doc.id
+
+        obj_data = {key: value if value is not None else None
+                    for key, value in obj_data.items()}
```

Comment:
:question: Do we need this if else here?
```suggestion
        obj_data = {key: value for key, value in obj_data.items()}
```